### PR TITLE
first program has done

### PR DESCRIPTION
**Pull request by Nodirjn**

### CODE REVIEW

File: first_program/first_program/main.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#include<iostream>
+
+using namespace std;
+
+int main()
+{
+     cout << "We started arithmetic operation!" << endl;
+    char operation;
```
Comment (farruhha) [inline first_program/first_program/main.cpp]: Remove the extra space

File: first_program/first_program/main.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    cin >> operation;
+    
+   
+    cout << "The first variable value: " << firstVariable << endl;
+    cout << "The second variable value: " << secondVariable << endl;
+    cout << "The operation: " << operation << endl;
+    
+    switch(operation){
+        case '+':
+             result=firstVariable+secondVariable;
```
Comment (farruhha) [inline first_program/first_program/main.cpp]: Fix the formatting by removing the extra space.

File: first_program/first_program/main.cpp [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            cout<<"Result of addition:"<<firstVariable<< "+" <<secondVariable<<"="<<result<<endl;
+            break;
+            
+        case '-':
+          result=firstVariable-secondVariable;
```
Comment (farruhha) [inline first_program/first_program/main.cpp]: Pay attention to formatting of your code and keep it consistent.

File: first_program/first_program/main.cpp
Comment (farruhha) [inline first_program/first_program/main.cpp:1]: @Nodirjn 

Approving your code changes for merging into the main but before you merge them please check the formatting of your code by removing the extra space or adding the extra space. Keep the code readable and consistent.

**Review by farruhha — APPROVED** [review]

**Review by farruhha — APPROVED** [review]

File: first_program/first_program/main.cpp (hunk 1 above)
Comment (Nodirjn) [reply first_program/first_program/main.cpp]: ookk

File: first_program/first_program/main.cpp (hunk 2 above)
Comment (Nodirjn) [reply first_program/first_program/main.cpp]: I have removed every extra space

**Review by farruhha — APPROVED** [review]

**Review by farruhha — APPROVED** [review]

**Review by farruhha — APPROVED** [review]

**Comment by farruhha** [comment]
You can merge it now

**Review by farruhha — APPROVED** [review]

**Review by farruhha — APPROVED** [review]

**Review by farruhha — APPROVED** [review]

**Review by farruhha — APPROVED** [review]

**Review by farruhha — APPROVED** [review]

**Review by farruhha — APPROVED** [review]

**Review by farruhha — APPROVED** [review]

**Pull request merged**